src/features/steps/3b-deti/deti.tsx

```diff
@@ -0,0 +1,284 @@
+import React from 'react'
```

Comment:
```
src/features/steps/3b-deti/deti.tsx
```
na to aby som pridal dalsi step za 3b musim vytvorit 3c a vsetky nasledujuce stepy zbytocne premenovat

Reply:
suhlas, toto bola docasna pomocka, ale rovnaky problem budes mat keby to mas ocislovane, takze moc s tymto nevyriesis, ale pomocka ktora ti poradi ze ako suvisia tie pages je velke plus na orientaciu v kode pri krokoch, a zase jednorazove premenovanie veci v jednom commite nikoho nezabije - rad sa pobavim ako to riesit lepsie, ale teraz by som to nebral tak probematicky